### PR TITLE
Adds script integration test, update fixture

### PR DESCRIPTION
**Pull request by zacharym**
@jsingle @thieman There are a number of things I'd like some advice on. Is the fixture too specific? Basically I needed an image I knew would have bash but that is just some randos. Also this is a pretty hacky way to get the docker binary. The issue is that running through the shell, docker can't properly parse the command, but running not through the shell, the `docker` doesn't find the binary.

### CODE REVIEW

File: tests/testcases.py [hunk 1]
```diff
@@ -125,8 +125,9 @@ def stdout(self):
         return sys.stdout.getvalue()[self.stdout_start:].strip()
 
     def exec_docker_patch(self, *args):
-        args = ['docker'] + [a for a in args]
-        self.exec_docker_process = subprocess.Popen(args=args, stdout=subprocess.PIPE, env=get_docker_env(), shell=True)
+        docker_executable = subprocess.check_output('which docker', shell=True)[0:-1]
```
Comment (jsingle) [inline tests/testcases.py]: is `[0:-1]` to cut off newline?  I think `rstrip` is better

File: tests/integration/cli/scripts_test.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+import time
+
+import git
+from mock import patch
+
+from dusty import commands
+from ...fixtures import basic_specs_fixture
+from ...testcases import DustyIntegrationTestCase
+
+class TestRestartCLI(DustyIntegrationTestCase):
```
Comment (jsingle) [inline tests/integration/cli/scripts_test.py]: name this better

File: tests/integration/cli/scripts_test.py [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+            self.run_command('stop --rm')
+        except Exception:
+            pass
+        rmtree(self.fake_repo_location)
+        super(TestRestartCLI, self).tearDown()
+
+    def test_basic(self):
+        self.exec_in_container('appa', 'touch /app/a/zoo')
+        self.run_command('scripts appa example')
+        self.assertEqual(self.exec_docker_process.stdout.read(), 'README.md  tee_dusty_always_fn_pipe_file  zoo\r\n')
```
Comment (jsingle) [inline tests/integration/cli/scripts_test.py]: I feel like comparing the output of a `ls` command is brittle.  Is there a more reliable script we can run?

Comment (zacharym) [inline tests/integration/cli/scripts_test.py]: yeah, i'll just exec_in_container that it exists. That will save a lot of effort

File: tests/integration/cli/scripts_test.py [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+    def tearDown(self):
+        self.run_command('bundles deactivate bundle-a')
+        self.run_command('repos manage github.com/app/a')
+        try:
+            self.run_command('stop --rm')
+        except Exception:
+            pass
+        rmtree(self.fake_repo_location)
+        super(TestRestartCLI, self).tearDown()
```
Comment (jsingle) [inline tests/integration/cli/scripts_test.py]: class name

File: tests/integration/cli/scripts_test.py [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+import git
+from mock import patch
+
+from dusty import commands
+from ...fixtures import basic_specs_fixture
+from ...testcases import DustyIntegrationTestCase
+
+class TestScriptsCLI(DustyIntegrationTestCase):
+    def setUp(self):
+        super(TestRestartCLI, self).setUp()
```
Comment (jsingle) [inline tests/integration/cli/scripts_test.py]: class name

File: dusty/commands/utils.py [hunk 6]
```diff
@@ -8,6 +9,12 @@
 def _executable_path(executable_name):
     return subprocess.check_output(['which', executable_name]).strip()
 
+def exec_docker_options():
+    exec_options = '-i'
+    if sys.stdout.isatty():
```
Comment (jsingle) [inline dusty/commands/utils.py]: I'm curious what issues this option was creating

**Pull request merged**